appserver/tests/application/src/test/java/org/glassfish/main/test/app/connpool/SQLTraceListenerTest.java

```diff
@@ -80,8 +80,8 @@ public static void deployAll() throws IOException {
         assertThat(result, asadminOK());
         // add-library shouldn't require a restart anymore
 
-        DOMAIN_SETTINGS.backupDerbyPoolSettings();
-        DOMAIN_SETTINGS.setDerbyPoolEmbededded();
+        DERBYPOOL_BACKUP.restore();
```

Comment:
This call seems redundant.

Reply:
Aha, you are right, and this was my mistake, mhm ...